openassessment/xblock/static/js/src/studio/oa_container_item.js

```diff
@@ -92,7 +92,9 @@ OpenAssessment.Prompt.prototype = {
      **/
     getFieldValues: function() {
         var fields = {
-            description: this.description()
+            description: this.description(),
```

Comment:
please don't make a variable and use return statement instead of fields.

Reply:
Done